docs/README.md

```diff
@@ -460,6 +460,22 @@ Returns the diagnostic information associated with the most recently called func
 
 **Returns**: `Promise` when resolved returns `String` or the promise is rejected.
 
+## **Statement.asNumber(flag)**
```

Comment:
This function name is confusing to me as it would seem to imply you're trying to convert the statement object to a number. I suggest using a more descriptive name.

Reply:
@kadler , the function name keeps align with idb-connector and some customers have already started to use it. I suggest to keep it, or change it only in idb-pconnector.

Reply:
Right, I am not proposing changing idb-connector, but there are sufficient differences that I think using a different name in idb-pconnector is fine.

Reply:
>  I think using a different name in idb-pconnector is fine.

Yeah this should be fine.

As for a more descriptive name. how about `.returnNumericsAsNumber` ?

Reply:
What about `enableNumericTypeConversion`?

Reply:
I will update the PR with the name change.